lib/phoenix_html/safe.ex

```diff
@@ -70,4 +70,11 @@ end
 
 defimpl Phoenix.HTML.Safe, for: Tuple do
   def to_iodata({:safe, data}), do: data
```

Comment:
@josevalim Done! This makes the error more specific at least – not sure if it helps much if you make that mistake, but as you say, it makes it easier to just add a description if needed :) I'll keep an eye out.